src/UI/ledger/LedgerLoginContainer/AddressRow.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useEffect, useState, SyntheticEvent } from 'react';
 import classNames from 'classnames';
 
-import globalStyles from 'assets/sass/main.scss';
+import { Trim } from 'UI/Trim';
 import { FormatAmount } from 'UI/FormatAmount/FormatAmount';
-import { getTrimmedHash } from 'utils';
 import { getAccountBalance } from 'utils/account/getAccountBalance';
 import { getEgldLabel } from 'utils/network/getEgldLabel';
-import { WithClassnameType } from '../../types';
+
+import type { WithClassnameType } from 'UI/types';
```

Comment:
You should use relative import here. There is transpilation issue and you will not be able to see the type properties in hierarchy. Please check the other places.

Reply:
Updated. 👍